src/ipinfoWrapper.ts

```diff
@@ -96,6 +99,8 @@ export default class IPinfoWrapper {
                                 ipinfo.countryCode = ipinfo.country;
                                 ipinfo.country =
                                     this.countries[ipinfo.countryCode];
+                                ipinfo.isEU = 
+                                this.euCountries.includes(ipinfo.countryCode);
```

Comment:
Can we format this? Seems weird that it's on the same indentation level as the assignment above it.